generate_workflow.py

```diff
@@ -158,11 +158,7 @@ def get_steps(build, build_set):
                     },
                 },
                 {
-                    "name": "Register clang error/warning problem matcher",
-                    "run": 'echo "::add-matcher::.github/problem-matchers/clang-errors-warnings.json"'
-                },
-                {
-                    "name": "Boot Test",
```

Comment:
Is it easy to break this into 2 commits; one for the python script changes, one for regen'ing the workflow files?

Reply:
I can do that, sure. It seemed like it should go together to avoid having a state where the problem matcher got registered twice.